src/Category/Table/CategoryTableBuilder.php

```diff
@@ -1,11 +1,17 @@
 <?php namespace Visiosoft\CatsModule\Category\Table;
 
 use Anomaly\Streams\Platform\Ui\Table\TableBuilder;
+use Illuminate\Database\Eloquent\Builder;
 use Visiosoft\CatsModule\Category\Table\Handler\Delete;
 
 class CategoryTableBuilder extends TableBuilder
 {
 
+    public function onQuerying(Builder $query)
+    {
+        $query->whereNull('parent_category_id');
```

Comment:
please add if condition for subcategories

``
if(!request()->cat){
//move here
}
``